.gitignore

```diff
@@ -1,10 +1,11 @@
 praw.ini
 *.pyc
 .idea/*
-archiver.log
-transcribersofreddit.log
-ocr.log
+*.log
 *.swp
 *.swo
 *.egg-info
 .eggs
+.cache/
+.idea/
+test/__pycache__/
```

Comment:
My concern was accidentally committing the pycache dir in tor_core. Then it occurred to me that there wouldn't ever be a time when we would commit that directory anywhere.